params/params_test.go

```diff
@@ -19,9 +19,9 @@ var usernameUnmarshalTests = []struct {
 }{{
 	username: "user",
 }, {
-	username: "admin@idm",
+	username: "admin@candid",
```

Comment:
I'm not certain these should be changed like this

Reply:
It's a good question. We should talk about that, although it probably doesn't matter for the purposes of username unmarshaling, as "idm" is just an arbitrary string in that context. I've changed it back anyway.